client/src/views/devices/GreetDeviceModal.vue

```diff
@@ -82,39 +97,53 @@
               </div>
               <!-- right element: invite link, copy button, email button -->
               <div class="right-side">
-                <ion-card class="card">
-                  <ion-card-content class="card-content">
-                    <ion-text
-                      v-if="!linkCopiedToClipboard"
-                      class="card-content__text body"
-                    >
+                <div
+                  class="link-content"
+                  v-if="isLargeDisplay"
+                >
+                  <div
+                    class="link-content-card"
+                    v-if="!linkCopiedToClipboard"
+                  >
+                    <ion-text class="link-content-card__text body">
                       {{ greeter.invitationLink }}
                     </ion-text>
-                    <ion-text
-                      v-else
-                      class="card-content__text body copied"
-                    >
-                      {{ $msTranslate('DevicesPage.greet.subtitles.copiedToClipboard') }}
-                    </ion-text>
                     <ion-button
                       fill="clear"
                       size="small"
                       id="copy-link-btn"
                       @click="copyLink"
-                      v-if="!linkCopiedToClipboard"
                     >
                       <ion-icon
                         class="icon-copy"
                         :icon="copy"
                       />
                     </ion-button>
+                  </div>
+                  <ion-text
+                    v-else
+                    class="link-content__text body copied"
+                  >
+                    {{ $msTranslate('DevicesPage.greet.subtitles.copiedToClipboard') }}
+                  </ion-text>
+                </div>
+                <div
+                  class="link-content-small"
+                  v-else
+                >
+                  <ion-button
+                    id="copy-link-btn-small"
+                    :class="linkCopiedToClipboard ? 'copied' : ''"
```

Comment:
`:class="{ copied: linkCopiedToClipboard }"` is a bit prettier, it doesn't have an empty else.